packages/mask/package.json

```diff
@@ -97,6 +97,7 @@
     "millify": "^4.0.0",
     "next-tick": "^1.0.0",
     "opensea-js": "^1.2.7",
+    "pure-react-carousel": "^1.28.1",
```

Comment:
Nobody likes the carousel. https://shouldiuseacarousel.com/

Please recheck the design.

Reply:
PM was requiring carousel.

Reply:
I'm not blocking this, just want to let them know